LevelMeter/LevelMeterSource.h

```diff
@@ -99,8 +99,8 @@ class LevelMeterSource
         float getAvgRMS () const
         {
             if (rmsHistory.size() > 0)
-                return std::sqrtf (std::accumulate (rmsHistory.begin(), rmsHistory.end(), 0.0f) / rmsHistory.size());
-
+                return std::sqrt(std::accumulate (rmsHistory.begin(), rmsHistory.end(), 0.0f) / static_cast<float>(rmsHistory.size()));
+                
```

Comment:
But best to run `Projucer --trim-whitespace --lf <folder>` at the end to get rid of leftover spaces and tabs... ;-)